apps/test/unit/templates/rubrics/RubricContainerTest.jsx

```diff
@@ -231,4 +234,60 @@ describe('RubricContainer', () => {
       'Error submitting feedback to student.'
     );
   });
+
+  it('passes down aiUnderstanding and aiConfidence to the LearningGoal', async () => {
+    const mockFetch = sinon.stub(global, 'fetch');
+    const aiEvaluationsMock = [
+      {learning_goal_id: 2, understanding: 2, confidence: 70},
+    ];
+    mockFetch.returns(
+      Promise.resolve(new Response(JSON.stringify(aiEvaluationsMock)))
+    );
+
+    const wrapper = mount(
+      <RubricContainer
+        {...defaultProps}
+        studentLevelInfo={{
+          name: 'Grace Hopper',
+          timeSpent: 305,
+          lastAttempt: '1980-07-31T00:00:00.000Z',
+          attempts: 6,
+        }}
+      />
+    );
+
+    await act(async () => {
+      await Promise.resolve();
+    });
+    wrapper.update(); // Apply the state update
```

Comment:
Curious if the update is needed? (totally fine if it is -- I've always just found it's not needed as often as I think)

Reply:
It is needed.